tests/factories/reports/cht/delivery.js

```diff
@@ -0,0 +1,260 @@
+        }
+      }
+    ],
+    'r_baby_danger_sign_present_joined': 'no',
+    'r_baby_danger_sign_present_any': 'no'
+  },
+  'safe_postnatal_practices': {
+    'safe_postnatal_practice_1': '',
+    'safe_postnatal_practice_2': '',
+    'safe_postnatal_practice_3': '',
+    'safe_postnatal_practice_4': '',
+    'safe_postnatal_practice_5': ''
+  },
+  'pnc_visits': {
+    'who_note': '',
+    'pnc_visits_attended': 'none',
+    'pnc_visits_additional': '0',
+    'days': '0'
+  },
+  'summary': {
+    'r_submit_note': '',
+    'r_summary_details': '',
+    'r_patient_details': '',
+    'r_pregnancy_outcome': '',
+    'r_woman_condition': '',
+    'r_condition_well': '',
+    'r_delivery_details': '',
+    'r_delivery_date': '',
+    'r_delivery_place': '',
+    'r_babies_delivered_num': '',
+    'r_babies_deceased_num': '',
+    'r_pnc_visits': '',
+    'r_pnc_visits_completed': '',
+    'r_pnc_visit_none': '',
+    'r_pnc_visits_add': '',
+    'r_referrals': '',
+    'r_who_schedule_note': '',
+    'r_pnc_schedule_note': '',
+    'custom_translations': {
+      'custom_woman_label_translator': 'woman',
+      'custom_woman_label': 'the woman',
+      'custom_woman_start_label_translator': 'woman-start',
+      'custom_woman_start_label': 'Woman',
+      'delivery_place_label_translator': 'health_facility',
+      'delivery_place_label': 'Health facility',
+      'woman_death_place_label_translator': '',
+      'woman_death_place_label': ''
+    }
+  },
+  'data': {
+    '__woman_condition': 'alive_well',
+    '__fever': 'no',
+    '__severe_headache': 'no',
+    '__vaginal_bleeding': 'no',
+    '__vaginal_discharge': 'no',
+    '__convulsions': 'no',
+    '__has_danger_sign': 'no',
+    '__woman_death_date': '',
+    '__woman_death_place': '',
+    '__delivered_before_death': '',
+    '__woman_death_notes': '',
+    '__babies_delivered': '1',
+    '__babies_deceased': '0',
+    '__babies_alive': '1',
+    '__delivery_date': '2021-04-27',
+    '__delivery_place': 'health_facility',
+    '__delivery_place_other': '',
+    '__delivery_mode': 'vaginal',
+    '__delivery_conductor': '',
+    '__delivery_conductor_other': '',
+    '__pnc_visit_within_24_hrs': 'no',
+    '__pnc_visit_3_days': 'no',
+    '__pnc_visit_7_days': 'no',
+    '__pnc_visit_6_weeks': 'no',
+    '__pnc_visits_num_selected': '0',
+    '__pnc_visits_additional': '0',
+    'meta': {
+      '__patient_uuid': '091d10d9-2f5c-4965-9c20-96730c6fc300',
+      '__patient_id': '96854',
+      '__household_uuid': '8935e9d8-0b2a-5eb0-ae54-ad1377a60097',
+      '__source': 'contact',
+      '__source_id': '',
+      '__pregnancy_uuid': 'b618d88b-fae5-4ab7-8289-a0416bd7294d'
+    }
+  },
+  'meta': {
+    'instanceID': 'uuid:a6f1aa20-036c-42dd-9415-c462ccb71c04'
+  }
+};
+
+Factory.define('baseDelivery')
+  .sequence('_id',uuid.v4)
+  .attr('form', 'delivery')
+  .attr('type', 'data_record')
+  .attr('content_type', 'xml')
+  .attr('reported_date', Date.now())
+  .attr('contact', 'TODO')
```

Comment:
I know this is a draft, but there is a lot in this one so just a reminder: DO this TODO :)